docs/src/pages/blog/digital-ocean-app-platform.tsx

```diff
@@ -0,0 +1,215 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+/* eslint-disable import/no-unresolved */
+import React from 'react'
+import Layout from '@theme/Layout'
+// @ts-ignore
+import DigitalOceanBanner from '@site/static/img/digital-ocean-banner.webp'
+// @ts-ignore
+import DigitalOceanLogin from '@site/static/img/digital-ocean-login.webp'
+// @ts-ignore
+import DigitalOceanCreateApp from '@site/static/img/digital-ocean-create-app.webp'
+// @ts-ignore
+import DigitalOceanAppConfig1 from '@site/static/img/digital-ocean-app-config-1.png'
+// @ts-ignore
+import DigitalOceanAppConfig2 from '@site/static/img/digital-ocean-app-config-2.png'
+// @ts-ignore
+import DigitalOceanStaticSite from '@site/static/img/digital-ocean-static-site.png'
+// @ts-ignore
```

Comment:
Try adding a `react-app-env.d.ts` to the `docs/` directory. With that, we can avoid these `@ts-ignore`.

```ts
declare module '*.png'
declare module '*.svg'
declare module '*.jpeg'
declare module '*.jpg'
declare module '*.webp'
```

Reply:
Done, thanks!

Reply:
![image](https://github.com/user-attachments/assets/a0e42dab-30cf-483c-8e97-c2cee855b965)
![image](https://github.com/user-attachments/assets/8a073b07-7857-47d6-987f-67fedebe717a)